src/renderer/components/opds/Entry.tsx

```diff
@@ -7,21 +7,23 @@
 
 import * as React from "react";
 import { Link, RouteComponentProps, withRouter } from "react-router-dom";
+import { OpdsLinkView } from "readium-desktop/common/views/opds";
 import * as ArrowIcon from "readium-desktop/renderer/assets/icons/baseline-arrow_forward_ios-24px.svg";
 import * as styles from "readium-desktop/renderer/assets/styles/opds.css";
 import SVG from "readium-desktop/renderer/components/utils/SVG";
 import { IOpdsBrowse } from "readium-desktop/renderer/routing";
 import { buildOpdsBrowserRoute } from "readium-desktop/renderer/utils";
 
 interface EntryProps extends RouteComponentProps<IOpdsBrowse> {
-    level: number;
-    entry: any;
+    level?: number;
+    entry: OpdsLinkView;
 }
 
-export class Entry extends React.Component<EntryProps, undefined> {
+class Entry extends React.Component<EntryProps, undefined> {
     public render(): React.ReactElement<{}>  {
         const { entry } = this.props;
 
+        // why ? entry is required
```

Comment:
I agree that this conditional statement should be removed.
Rationale: in principle, better to see a "crash" because of `undefined` `entry` in the following `entry.url` line of code, rather than silently ignoring the (impossible) occurrence of undefined `entry: OpdsLinkView`.

Reply:
9f61dd0